06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md

```diff
@@ -0,0 +1 @@
+[pull request](https://github.com/andela-aalabi/checkpoint1/pull/1)
```

Comment:
I see the link to your repo. This Readme is not properly done. Please see your peers for the proper format.